plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/niassa/WorkflowAction.java

```diff
@@ -186,12 +189,15 @@ public final ActionState perform(ActionServices actionServices) {
       // We get exactly one attempt to launch a job. If we fail, that's the end of this action. A
       // human needs to rescue us.
       if (hasLaunched) {
+        this.errors =
+            Collections.singletonList("Workflow has already been attempted. Purge to try again.");
         return ActionState.FAILED;
       }
 
       // Check if there are already too many copies of this workflow running; if so, wait until
       // later.
       if (server.get().maxInFlight(workflowAccession)) {
+        this.errors = Collections.singletonList("Too many workflows running.");
```

Comment:
Could this be clarified to indicate that this will resolve without user action? `Workflow has not yet launched because too many copies of this workflow are running. New workflows will be launched as current workflows complete.` or similar?